src/solver/conv_MP_bidirectional_winograd.cpp

```diff
@@ -31,22 +31,33 @@
 #include <miopen/gcn_asm_utils.hpp>
 #include <miopen/tensor.hpp>
 #include <miopen/gemm_v2.hpp>
+#include <miopen/stringutils.hpp>
+#include <miopen/miopen.h>
+#include <miopen/generic_search.hpp>
+#include <miopen/conv/invokers/impl_gemm.hpp>
 
 #include <boost/any.hpp>
-
-#if(MIOPEN_BACKEND_HIP && MIOPEN_USE_ROCBLAS)
 #include <miopen/conv/data_invoke_params.hpp>
+
+#if(MIOPEN_BACKEND_HIP)
+
 #define WORKAROUND_SWDEV_203031 1 // See also issues #2075, #2067
 #endif
 
 namespace miopen {
 namespace solver {
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F2X3)
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X3)
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F4X3)
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F5X3)
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F6X3)
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_WORKSPACE_MAX)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_MP_BD_WINOGRAD_F2X3)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_MP_BD_WINOGRAD_F3X3)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_MP_BD_WINOGRAD_F4X3)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_MP_BD_WINOGRAD_F5X3)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_MP_BD_WINOGRAD_F6X3)
```

Comment:
Do we really need to rename envvars? If so, then we have to update the documentation. Also all the users who use these envvars would need to update theirs scripts.

Reply:
Without renaming 1 envvar will manage 3 solvers
ConvMPBidirectWinograd
ConvMPBidirectWinograd_xdlops
ConvWinograd3x3MultipassWrW

Reply:
Formally speaking, no. The old names would look ambiguous without renaming, but this is acceptable cost of the backward compatibility. Thanks for fixing.